src/Raygun4php/RaygunClient.php

```diff
@@ -129,17 +129,17 @@ public function SendError(
     /**
      * Transmits an exception to the Raygun.io API
      *
-     * @param \Exception $exception      An exception object to transmit
+     * @param \Throwable $throwable      An exception object to transmit
```

Comment:
Exception implements Throwable from PHP 7 onwards. Perhaps for the sake of PHP 5.6 (static analysis) we could hint both for now? `@param \Exception|\Throwable $throwable`

Reply:
Technically according to the `composer.json` the minimum requirement is PHP 7.1, so is this an issue?

Reply:
No. Cheers!